### PR TITLE
Adding a note about Android Studio missing the Tools>Android submenu.

### PR DESCRIPTION
**Pull request by Sfshaza**
Fixes https://github.com/flutter/flutter/issues/14624

Please review, @mit-mit

### CODE REVIEW

File: _includes/setup/android-setup.md [hunk 1]
```diff
@@ -39,11 +39,13 @@ To prepare to run and test your Flutter app on the Android emulator, follow thes
 
 1. Enable [VM acceleration](https://developer.android.com/studio/run/emulator-acceleration.html) on your machine.
 1. Launch **Android Studio>Tools>Android>AVD Manager** and select **Create Virtual Device**.
+(If the **Tools** menu has no **Android** submenu, see
+[StackOverflow](https://stackoverflow.com/questions/40407124/android-studio-2-2-2-tools-android-menu-is-missing-in-mac-os-x.)
```
Comment (mit-mit) [inline _includes/setup/android-setup.md]: As mentioned over in the bug I don't think this SO answer is actually what's going on

Comment (Sfshaza) [reply _includes/setup/android-setup.md]: It seemed from that SO post, that it could be one of several things. I'm not sure where to point them.

Comment (mit-mit) [reply _includes/setup/android-setup.md]: I would just add a note that the menu entry is only present when inside an Android project. As noted in the bug this is all hopefully temporary anyway.

**Comment by Sfshaza** [comment]
@mit-mit, can you review?

File: _includes/setup/android-setup.md [hunk 2]
```diff
@@ -38,12 +38,14 @@ you want Flutter to use a different installation of the Android SDK, you must se
 To prepare to run and test your Flutter app on the Android emulator, follow these steps:
 
 1. Enable [VM acceleration](https://developer.android.com/studio/run/emulator-acceleration.html) on your machine.
-1. Launch **Android Studio>Tools>Android>AVD Manager** and select **Create Virtual Device**.
+1. Launch **Android Studio>Tools>Android>AVD Manager** and select
```
Comment (devoncarew) [inline _includes/setup/android-setup.md]: Note that I do not see this anymore; in Android Studio 3.1 (the current stable), I see the `AVD Manager` entry directly off the `Tools` menu (not in a submenu), and it does show for Flutter projects as well as regular Android ones.

Comment (devoncarew) [reply _includes/setup/android-setup.md]: Similarly in Android 3.2 Beta.

**Pull request merged**